File: 프로그래머스/1/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.py
def solution(T, P):
    t = []
    p = int(P)
    for i in range(0,len(T)-(len(P)-1)):
        answer = ""
        for j in range(i, i+len(P)):
            answer += T[j]
        t.append(int(answer))
    answer = [x for x in (t) if x <= p]
    
    return len(answer)

File: 프로그래머스/1/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.swift
import Foundation

func solution(_ t:String, _ p:String) -> Int {
    var answer : Int = 0
    var index : Int = 0
    var array : [String] = []
    let stringToArray : Array = Array(t)
    var result : String = ""
    while true {
        if index + p.count - 1 >= stringToArray.count {
            break
        }
        for char in index..<index + p.count {
            result += (String(stringToArray[char]))
        }
        array.append(result)
        index += 1
        result = ""
    }
    print(array)
    for arr in array {
        if p >= arr {
            answer += 1
        }
    }
    return answer
}